cloudbuild/unit_test_jobs.jsonnet

```diff
@@ -34,7 +34,7 @@ local unittest = base.BaseTest {
     |||
   ],
   command: [
-    'pytest --ignore keras_cv/models/legacy/',
+    'pytest --ignore keras_cv/models/legacy/ --durations 0',
```

Comment:
This is a nice feature -- didn't know this existed

Reply:
Yes, and this way we can easily check in on slow tests from the GCB logs